src/utils.jl

```diff
@@ -1,9 +1,8 @@
 """
+sample!([rng,] a, k; exclude = ())
+
```

Comment:
I think that is denoting rng is an optional argument. There is a second method of this function that does not take rng as an argument and uses the standard one.